app/views/templates/layout.scala.html

```diff
@@ -0,0 +1,98 @@
+@import uk.gov.hmrc.hmrcfrontend.views.Aliases.Banner
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.language.Cy
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.language.En
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.timeoutdialog.TimeoutDialog
+
+@this(
+    trackingConsentSnippet: HmrcTrackingConsentSnippet,
+    hmrcTimeoutDialog: HmrcTimeoutDialog,
+    hmrcBanner: HmrcBanner,
+    phaseBanner: components.phaseBanner,
+    govukBackLink: GovukBackLink,
+    hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper,
+    siteHeader: components.siteHeader,
+    govukLayout: GovukLayout,
+    hmrcStandardFooter: HmrcStandardFooter
+)
+
+@(pageTitle: Option[String] = None, beforeContentBlock: Option[Html] = None, backLink: Boolean = true, signOutLink: Boolean = true, timeout: Boolean = true)(contentBlock: Html)(implicit request: Request[_], messages: Messages, appConfig: config.AppConfig)
+
+@head = {
+<!--[if lte IE 8]><link href=' @controllers.routes.Assets.at("stylesheets/application-ie-8.css")' rel="stylesheet" type="text/css" /><![endif]-->
+<!--[if gt IE 8]><!--> <link href='@controllers.routes.Assets.at("stylesheets/application.css")' media="screen" rel="stylesheet" type="text/css" />
+<!--<![endif]-->
+@trackingConsentSnippet()
+@if(timeout) {
+    @hmrcTimeoutDialog(TimeoutDialog(
+        language = Some("en"),
+        timeout = Option(appConfig.timeoutInSecondsInt),
+        countdown = Option(appConfig.timeoutDisplayLengthInt),
+        keepAliveUrl = Some(controllers.reg.routes.SignInOutController.renewSession.url),
+        signOutUrl = Some(controllers.reg.routes.SignInOutController.destroySession.url)
+    ))
+}
+}
+
+@beforeContentBlock = {
+@phaseBanner("beta")
+
+@if(backLink == true) {
+@govukBackLink(BackLink(href = "#", content = Text(messages("site.back")), attributes = Map("id" -> "back-link")))
+}
+}
+
+@scripts = {
+<script src='@controllers.routes.Assets.at("lib/govuk-frontend/govuk/all.js")'></script>
+<script src='@controllers.routes.Assets.at("lib/hmrc-frontend/hmrc/all.js")'></script>
+<script>window.GOVUKFrontend.initAll();
+    window.HMRCFrontend.initAll()</script>
+<script src="@controllers.routes.Assets.at("javascripts/backLink.js")"></script>
+}
+
+@content = {
+    <div class="govuk-!-margin-bottom-6">
+       @contentBlock
+    </div>
+
+    <div class="govuk-body">
+        <a lang="en" hreflang="en" class="govuk-link " target="_blank" href="https://www.tax.service.gov.uk/contact/report-technical-problem?service=SCRS">
+        @messages("app.getHelp")
+        </a>
+    </div>
+}
+
+@govukLayout(
+    pageTitle = pageTitle,
+    headBlock = Some(head),
+    beforeContentBlock = Some(beforeContentBlock),
+    bodyEndBlock = None,
+    headerBlock = Some(siteHeader(signOutLink)),
+    footerBlock = Some(hmrcStandardFooter(accessibilityStatementUrl = Some(appConfig.accessibilityStatementUrl(request.uri)))),
+    scriptsBlock = Some(scripts)
+)(content)
+
```

Comment:
Can this be updated to the HMRC Layout. Works a lot better and you can get rid of a lot of the scripts and hardcoded html. Would need doing for the Welsh eventually anyway. PAYE-FE has an example